tests/globus_test.py

```diff
@@ -215,6 +215,45 @@ def test_file_rename_with_ftp(self):
                 ftp.login(user='user1', passwd='pass')
                 ftp.delete(f'/tempZone/home/user1/{filename2}')
 
+    def test_preserve_file_modification_time(self):
+        filename = f'{inspect.currentframe().f_code.co_name}1'
+        modify_time = '19710827010000'
```

Comment:
What is the unit of time for this value?

Does this value result in the expected timestamp appearing in iRODS?
It's probably a good idea to add an assertion for this using the icommands _(assuming the icommands are available here)_.

Reply:
This time is in format YYYYMMDDHHMISS.  That is just the way it is done.

I can't at the moment test it with icommands because I don't have the ability to run icommands as the same user as the anonymous FTP.  There is an enhancement open for adding the ability to run icommands for other users (similar to core tests).

Reply:
Cool cool.

Reply:
ISO8601 usually has a `T` between the date and the time part.   `YYYYMMDDThhmmss`

is this human visible/parsable/usable by others?  or just the test itself?